repos/system_upgrade/el7toel8/libraries/overlaygen.py

```diff
@@ -17,7 +24,12 @@ def _ensure_enough_diskimage_space(space_needed, directory):
 
 
 def _get_mountpoints(storage_info):
-    return list(set([entry.fs_file for entry in storage_info.fstab if os.path.isdir(entry.fs_file)]))
+    mount_points = set()
+    for entry in storage_info.fstab:
+        if os.path.isdir(entry.fs_file) and entry.fs_vfstype not in OVERLAY_DO_NOT_MOUNT:
+            mount_points.add(MountPoints(entry.fs_file, entry.fs_vfstype))
+
```

Comment:
@Rezney I am ok to merge it as it is, but what do you think about printing a warning when we skip vfat? it can be done even in the following PR if you want, so we will be able to merge this already.

Reply:
Ok, will do...